clm/mlperf_logging_utils.py

```diff
@@ -153,6 +163,22 @@ def on_step_end(
         Event called at the end of a training step.
         """
         if state.global_step % (state.eval_steps) == 0 and state.global_step > 0:
+            self.mllogger.event(
+                "train_loss",
+                value=state.log_history[-1]["loss"] if state.log_history else -1,
```

Comment:
Is it possible to use `train/loss` instead to make it different to eval one.
Instead of `loss`, torch/xla use the key `train/loss` for both mllog and tensorboard tracking as well.

Let me know what you think. If it is not easy to change in nemo side. I can follow your practice and add a fix after merging this change.